src/modsim2/attack/adversarial_images.py

```diff
@@ -148,6 +154,9 @@ def generate_adversarial_images(
                         L2FastGradientAttack or BoundaryAttack
         epsilons: Pertubation parameter for the attacks
         device: String passed to fb.PyTorchModel for computation
+        batch_size: int,
+        trainer_kwargs: dict = {},
+        loss_function: Callable = torch.nn.functional.nll_loss,
         **kwargs: Additional arguments based to attack setup
 
     Returns: a torch.tensor containing the adversarial images and a torch.tensor
```

Comment:
I think this statement needs a small update to describe the vulnerability metric?

Reply:
Have updated